lib/RESTClient.js

```diff
@@ -419,24 +419,25 @@ class RESTClient {
     *   @param {string} raftId - raft session id
     *   @param {number} [start=undefined] - starting sequence number. If it is
     *       not given, its value will be 1
-    *   @param {number} end=undefined] - ending sequence number. If it is not
-    *       given, 10K entries is limited to be retrieved
+    *   @param {number} [limit=undefined] - maximum number of log records
+    *       to return. It is at most of 10K. If it is not given, max 10K logs
+    *       would be return.
     *   @param {boolean} [targetLeader=undefined] - true: from leader instead of
     *       follower
     *   @param {string} reqUids - the identifier of the request
     *   @param {callback} callback - callback(err, info)
     *   @param {werelogs.Logger} [logger] - Logger instance
     *   @return {undefined}
     */
-    getRaftLog(raftId, start, end, targetLeader, reqUids, callback, logger) {
+    getRaftLog(raftId, start, limit, targetLeader, reqUids, callback, logger) {
         const log = logger || this.createLogger(reqUids);
         const path = `/_/raft_sessions/${raftId}/log`;
         const query = {};
         if (start !== undefined && start !== null) {
             query.begin = start;
         }
-        if (end !== undefined && end !== null) {
-            query.end = end;
+        if (limit !== undefined && limit !== null) {
+            query.limit = limit;
```

Comment:
This means changing the protocol. Not an issue since we don't have a constraint of backwards compatibility for this feature; but let's be careful anyways.